lib/Feature/FeatureModelParser.cpp

```diff
@@ -30,7 +30,9 @@ bool FeatureModelXmlParser::parseConfigurationOption(xmlNode *Node,
       } else if (!xmlStrcmp(Head->name, XmlConstants::OPTIONAL)) {
         Opt = Cnt == "True";
       } else if (!xmlStrcmp(Head->name, XmlConstants::PARENT)) {
-        FMB.addParent(Name, Cnt);
+        // TODO(s9latimm) fix me -- breaks if Name not set / parsed yet
```

Comment:
What should we do in this case?

Reply:
We could search `name`  beforehand with an initial pass over all `Node->children`, unfortunately that way we have to look at each node twice (or at least until we find `name`). I will add a separate PR from `vara-dev` with the fix, so that we can resolve this quickly.